### DISCUSSION TITLE
Add equivalent style to place=square as in leisure=park

### DISCUSSION THREAD

**Discussion by patepelo** [organicmaps/organicmaps#6321] [Ideas]
- Font could be green and white as well
- We could add an equivalent (not circled) icon

![image](https://github.com/organicmaps/organicmaps/assets/23003127/eb9a09e4-0401-44be-a5e6-1931ff898ae5)

**Comment by hb0nd** [comment]
squares aren't always grassy, in London the majority are concrete, so green wouldn't make sense IMO.
i'll try adding an outline to the standard grey and post screenshots

**Reply by patepelo** [reply]
OK. Just grey should work, at least is much better. And you're right that squares tend to be grey.

**Comment by hb0nd** [comment]
also we decided against an icon in https://github.com/organicmaps/organicmaps/issues/6013 :)

PR with photos here:
https://github.com/organicmaps/organicmaps/pull/6322

**Reply by patepelo** [reply]
It looks much better. Thank you for taking it into account. I think it makes a great difference.
